velox/docs/functions/spark/string.rst

```diff
@@ -179,6 +179,14 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: substring_index(string, string, count) -> varchar
```

Comment:
Can we distinguish the variables of this function signature? E.g., substring_index(string, delim, count).

Reply:
Done.